scripts/getting-started.sh

```diff
@@ -114,8 +114,16 @@ if command -v rustc >/dev/null 2>&1; then
     echo "\n✅︎🦀 Rust already installed."
 else
     if prompt_default_yes "\n🦀 Rust is not installed. Install it?"; then
-        echo "🦀 Installing via rustup."
-        curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+        version=`cat ../.github/env | grep IMAGE | cut -d'-' -f3`
```

Comment:
This seems flaky to me, any change to the other file will break this. 

I think the nicer thing to do is to store this in a nicely discover-able variable, and have it be used in both places.

Reply:
I would expect @alvicsam to know if my worry here is real or not, perhaps it is fine.

Reply:
It's fine as a temporary solution for now, in the future it should be switched to using `rust-toolchain.toml`

Reply:
> in the future it should be switched to using rust-toolchain.toml



is this tracked in an issue?

Reply:
I saw a [comment](https://github.com/paritytech/polkadot-sdk/issues/5263#issuecomment-2289051114) from @alvicsam regarding adding the version to the `rust-toolchain.toml`. We can definitely parse the version from there once the CI migration merges.

Reply:
yep, here: https://github.com/paritytech/polkadot-sdk/issues/5335